pkgs/applications/networking/newsreaders/slrn/default.nix

```diff
@@ -1,14 +1,13 @@
 { stdenv, fetchurl
 , slang, ncurses, openssl }:
 
-let version = "1.0.2"; in
-
 stdenv.mkDerivation {
```

Comment:
```suggestion
stdenv.mkDerivation rec {
```

Reply:
otherwise version in src.url can't find it.

Reply:
Ah, good catch. Thanks.